add keyfacts on environment overview

DaveDarsa opened the pull request.

CGoodwin90 reviewed (commented):
Looking good code wise! Just noticed a couple of items, otherwise all good.

The first `Programming language` fact looks like a fallback, is that a duplicate of `language` and is it neccessary to display?

![image](https://github.com/user-attachments/assets/a27257b7-3956-4b22-a2bc-b8a26fec1bdf)

Also is the `Keyfacts` component nested between the environment details and lower buttons by design? Just noticed it looks a little off on environments with active/standby.

![Screenshot from 2025-05-07 09-33-51](https://github.com/user-attachments/assets/32a6cf48-10cd-40ec-a997-14cc19c82a1d)

DaveDarsa commented:
@CGoodwin90 good point! - I added a check so the categories shown here mirror the ones in the keyfact matrix doc:

- `OS`
- `Lagoon`
- `Service`
- `Language`
- `Application`
- `Helper`

As for the the location, the design is ripped from UI 2 - between environment details and action buttons, but I agree with button margins, so I just added the same margin to the active/standby that the delete one has.

rocketeerbkw commented:
Tested and working as intended in another communication channel.
